src/middlewared/middlewared/plugins/failover_/journal.py

```diff
@@ -118,7 +118,8 @@ def _handle_sql_queue_item(self, item):
                 self.journal.append(item)
             else:
                 query, params = item
-                logger.warning('Node status %s but executed SQL query: %s', self.failover_status, query)
+                if query and query.find('services_ssh') == -1:
```

Comment:
Can't you just use `ha_sync: false` for the update host keys query?

Reply:
I've reverted that change and will add it to my open PR here: https://github.com/truenas/middleware/pull/7843